### PR TITLE
Fix extensions breaking change in blackbox tests

### PR DESCRIPTION
**Pull request by licitdev**
## Scope

What's changed:

- Fix extensions breaking change in blackbox tests

## Potential Risks / Drawbacks

- None

## Review Notes / Questions

- None

---

Related to #20502

### CODE REVIEW

File: tests/blackbox/extensions/action-verify-create/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "action-verify-create",
+	"version": "0.0.0",
+	"private": true,
+	"keywords": [
+		"directus",
+		"directus-extension",
+		"directus-custom-hook"
+	],
```
Comment (paescuj) [inline tests/blackbox/extensions/action-verify-create/package.json:9]: Let's reduce to the props that are actually required, so we don't have to keep these files in "sync"

```suggestion
```

File: tests/blackbox/extensions/action-verify-schema/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "action-verify-schema",
+	"version": "0.0.0",
+	"private": true,
+	"keywords": [
+		"directus",
+		"directus-extension",
+		"directus-custom-hook"
+	],
```
Comment (paescuj) [inline tests/blackbox/extensions/action-verify-schema/package.json:9]:
```suggestion
```

File: tests/blackbox/extensions/action-verify-schema/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+		"build": "directus-extension build",
+		"dev": "directus-extension build -w --no-minify",
+		"link": "directus-extension link"
+	},
+	"devDependencies": {
+		"@directus/extensions-sdk": "11.0.2"
+	},
+	"files": [
+		"*"
+	]
```
Comment (paescuj) [inline tests/blackbox/extensions/action-verify-schema/package.json:26]:
```suggestion
	}
```

File: tests/blackbox/extensions/action-verify-create/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+		"build": "directus-extension build",
+		"dev": "directus-extension build -w --no-minify",
+		"link": "directus-extension link"
+	},
+	"devDependencies": {
+		"@directus/extensions-sdk": "11.0.2"
+	},
+	"files": [
+		"*"
+	]
```
Comment (paescuj) [inline tests/blackbox/extensions/action-verify-create/package.json:26]:
```suggestion
	}
```

**Review by paescuj — APPROVED** [review]
❤️

File: tests/blackbox/extensions/action-verify-create/package.json (hunk 1 above)
Comment (licitdev) [reply tests/blackbox/extensions/action-verify-create/package.json]: Very true! Thanks!

**Pull request merged**